index.html

```diff
@@ -42,6 +42,223 @@ <h2 class="connect-link">LET’S CONNECT</h2>
+                    </li>
+                    <li><a href="Github.com"><img class="logo-images" src="./images/Github.svg" alt="Github logo"></a>
+                    </li>
+                    <li><a href="angelist.com"><img class="logo-images" src="./images/Angelist.svg"
+                                alt="Angelist logo"></a>
+                    </li>
+                </ul>
+                <div>
+                    <ul class="lang-list">
+                        <li class="language">Languages</li>
+                        <li><img src="./images/Down-arrow.png" alt="Down-arrow"></li>
+                    </ul>
+                    <ul class="lange-type">
+                        <li class="lange-type-logo"><img src="./images/js-lang-logo.png" alt="Javascript-logo">
+                            <span>Javascript</span>
+                        </li>
+                        <li class="lange-type-logo"><img src="./images/html5-logo.png" alt="html-logo">
+                            <span>HTML</span>
+                        </li>
+                        <li class="lange-type-logo"><img src="./images/css-lang-logo.png" alt="css-logo">
+                            <span>CSS</span>
+                        </li>
+                    </ul>
+                    <ul class="lang-list-frame">
+                        <li class="language">Frameworks</li>
+                        <li><img src="./images/Right-arrow.png" alt="Down-arrow"></li>
+                    </ul>
+                    <ul class="lang-list-frame">
+                        <li class="language">Skills</li>
+                        <li><img src="./images/Right-arrow.png" alt="Down-arrow"></li>
+                    </ul>
+                </div>
+            </article>
+        </section>
+        <section class="contact-container">
+            <div class="contact-container-sub">
+                <h2 class="contact-h2">Contact me</h2>
+                <p class="contact-info">If you have an application you are interested in developing, a feature
+                    that you need built or a project that needs coding. I’d love to help with it</p>
+            </div>
+            <article class="form-article">
+                <form action="https://formspree.io/f/xpzcccob" method="POST" class="contact-form" id="contact-form">
+                    <label for="name"></label>
+                    <input type="text" name="Name" placeholder="Write name." required maxlength="30" id="name">
+                    <label for="email"></label>
+                    <input type="email" name="Email" placeholder="example@gmail.com" required id="email">
+                    <label for="message-container"></label>
+                    <textarea name="message" id="message-box" cols="30" rows="10"
+                        placeholder="write your message in here" maxlength="500" required></textarea>
+                    <button type="submit" class="get-in-touch-btn
+                    ">Get It Touch</button>
```

Comment:
- [ ] Please make sure that your submit button is aligned together / is in line with the form.
-   Your design
     ![portfo2](https://user-images.githubusercontent.com/87394385/212286498-22bb91b4-0dd3-4314-97e6-0fcc7c999970.JPG)

-   Figma design
     ![portfo3](https://user-images.githubusercontent.com/87394385/212286636-cece4bc3-f4f0-4bf9-bb14-2be427efdc55.JPG)